File: APIflow/python/tutorials/listTasks.py
import sys, getopt

exec(open("../APIFlow.py").read())

email, password, apiKey = sys.argv[1], sys.argv[2], sys.argv[3]

sessionKey = auth(email, password, apiKey)
tasks = listTasks(sessionKey)

#review my tasks
for t in tasks:
	print(t)

#complete tasks with empty json body
# completeTask(sessionKey, t["ID"])

File: APIflow/ruby/tutorials/listTasks.rb
require_relative '../APIFlow.rb'

email, password, apiKey = ARGV[0], ARGV[1], ARGV[2]

flow = APIFlow.new(email, password, apiKey)

sessionKey = flow.sessionKey()
tasks = flow.listTasks(sessionKey)

#review my tasks
for t in tasks
	print(t)
end

#complete tasks with empty body
# flow.completeTask(sessionKey, t["ID"])